frontend/public/locale/es/translation.json

```
@@ -0,0 +1,22 @@
+{
+  "homekey": "Inicio",
+  "aboutkey": "Acerca de",
+  "contactkey": "Contacto",
```

Biome (1.9.4)

[error] The key contactkey was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)